packages/rs-drive-abci/src/execution/engine/run_block_proposal/v0/mod.rs

```diff
@@ -288,7 +288,7 @@ where
 
         // while we have the state transitions executed, we now need to process the block fees
 
-        let block_fees_v0: BlockFeesV0 = block_fees.into();
+        let block_fees_v0: BlockFeesV0 = state_transitions_result.aggregated_fees().clone().into();
```

Comment:
Not a great idea to clone here, best to have the into method also be able to take a borrowed object.

Reply:
We can't do move fees from `state_transitions_result` since we need it below

Reply:
What I am saying is that an into() can clone exactly what you want, instead of cloning the entire object first.

Reply:
This doesn't matter that much to me.